apps-rendering/src/components/LiveBlocks/index.tsx

```diff
@@ -29,6 +31,12 @@ const LiveBlocks: FC<LiveBlocksProps> = ({
 	const showPinnedPost =
 		pageNumber === 1 && pinnedPost.kind === OptionKind.Some;
 
+	const showAd = (index: number): boolean =>
+		// This can be removed when LiveBlogs are deployed
+		format.design === ArticleDesign.DeadBlog &&
+		// Add an AdSlot at 2nd and every other 5 blocks
+		(index === 1 || (index - 1) % 5 === 0);
```

Comment:
coming back to this again today, I found it hard to figure out what this was doing (even with the comment!). I wonder if it would help to give a name to the value `1` here, so the calculations are easier to read. I thought of something like `firstAdOffset` but maybe there's a better option! What do you think?

```suggestion
		(index === firstAdOffset || (index - firstAdOffset) % 5 === 0);
```